src/main/java/com/xpanxion/java/springboot/da1/demo/controller/student10/MessageController10.java

```diff
@@ -8,6 +8,6 @@
 
     @GetMapping("student10/api/v1/message")
     public String getMessage() {
-        return "My name is Rachel K Eckels and my favorite color is green and my favorite band is Birdtalker.";
+        return "My name is Rachel Eckels and my favorite color is green and my favorite band is Birdtalker.";
```

Comment:
This PR reverts the middle initial from Ex 3.  Is there a reason for this?  Can we add the middle initial back?

PRs should focus on one thing and have a very narrow focus.  The title of this PR states that this is for Ex6 but we are also reverting Ex 3, which is not ideal.

Reply:
I had changed it yesterday to test something. I'll change it back though!